pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -353,23 +353,15 @@ func (state PipelineRunState) getNextTasks(candidateTasks sets.String) []*Resolv
 // getRetryableTasks returns a list of pipelinetasks which should be executed next when the pipelinerun is stopping,
 // i.e. a list of failed pipelinetasks from candidateTasks which haven't exhausted their retries. Note that if a
 // pipelinetask is cancelled, the retries are not exhausted - they are not retryable.
+//
+// Note: for now, this function only detects if v1alpha1.Run is retriable. For v1beta1.CustomRun and TaskRuns,
+// the retries implementation is hiden from PipelineRun reconciler. See TEP-0121 for details.
 func (state PipelineRunState) getRetryableTasks(candidateTasks sets.String) []*ResolvedPipelineTask {
```

Comment:
to be specific, let's rename this function to `getRetryableRuns` or something similar

Reply:
SG, Updated.